src/components/brew-input.js

```diff
@@ -1,29 +1,33 @@
 import React from 'react';
 import {connect} from 'react-redux';
+import moment from 'moment';
 
 import {addBrew} from '../database/brew';
 import {FancyInput, FancyDropdown, FancyButton} from './input-stuff.js'
 
 
 var testEquip = ["Chemex", "Clever", "French Press"];
-// Eventually this would be stored elsewhere, yes?
-
 
 const FirstPanel = (props) => {
   return (
     <div>
       <FancyInput label="Coffee Name " name="name" handleChange={props.handleChange}
         value={props.name}/>
-      <FancyDropdown label="Brew Method " name="brewMethod" options={testEquip}
-        handleChange={props.handleChange}/>
+      <FancyDropdown label="Brew Method " name="brewMethod" handleChange={props.handleChange}
+        options={testEquip}
+        />
       <FancyInput label="Brew Date" name="brewDate" handleChange={props.handleChange}
-        value ={props.brewDate} />
+        value={props.brewDate} />
+      <FancyButton name="dateButton"
+        handleSubmit={props.setDateToday}
```

Comment:
Should I have two props then in the `FancyButton` component in `input-stuff,` one `handleSubmit` for when its working with the submit event, and then `onClick` otherwise?